tdms/tests/unit/test_interp_determination.cpp

```diff
@@ -81,12 +91,15 @@ TEST_CASE("determineInterpScheme: correct interpolation chosen") {
     CHECK(best_interp_scheme(N, 1).get_priority() == BAND_LIMITED_0);
     CHECK(best_interp_scheme(N, 2).get_priority() == BAND_LIMITED_1);
     CHECK(best_interp_scheme(N, 3).get_priority() == BAND_LIMITED_2);
-    for(int i=4; i<=N-4; i++) {CHECK(best_interp_scheme(N, i).get_priority() == BAND_LIMITED_3);}
-    CHECK(best_interp_scheme(N, N-3).get_priority() == BAND_LIMITED_4);
-    CHECK(best_interp_scheme(N, N-2).get_priority() == BAND_LIMITED_5);
-    CHECK(best_interp_scheme(N, N-1).get_priority() == BAND_LIMITED_6);
-
-    /* If 4<=N<=8 we can still fall back on cubic interpolation 
+    for (int i = 4; i <= N - 4; i++)
+    {
+        CHECK(best_interp_scheme(N, i).get_priority() == BAND_LIMITED_3);
+    }
```

Comment:
More trivial suggestions:
```suggestion
    for (int i = 4; i <= N - 4; i++)
        CHECK(best_interp_scheme(N, i).get_priority() == BAND_LIMITED_3);
```
Or maybe one line? (Whatever your preferred editor/linter suggests.)